src/js/idx-omnibar.js

```diff
@@ -515,7 +515,9 @@ var idxOmnibar = function(jsonData){
 						break;
 					case 'zipcodes':
 						foundResult = true;
-						goToResultsPage(input, idxUrl, '?pt=' + basicPtID + '&ccz=zipcode&zipcode[]=' + list[i].id + '&srt=' + sortOrder);
+						var zipString = `${list[i].id}`
+						var zipSearch = zipString.length == 3 ? '&ccz=zipcode&k_zipcode[]=' :  '&ccz=zipcode&zipcode[]='
```

Comment:
A comment here might be helpful to understand why a zipcode would be a length of 3